components/ActivityIndicator/src/MDCActivityIndicator.m

```diff
@@ -809,4 +814,73 @@ - (void)applyPropertiesWithoutAnimation:(void (^)(void))setPropBlock {
   [CATransaction commit];
 }
 
+#pragma mark - Resource Bundle
+
++ (NSBundle *)bundle {
+  static NSBundle *bundle = nil;
+  static dispatch_once_t onceToken;
+  dispatch_once(&onceToken, ^{
+    bundle = [NSBundle bundleWithPath:[self bundlePathWithName:kMaterialActivityIndicatorBundle]];
+  });
+
+  return bundle;
+}
+
++ (NSString *)bundlePathWithName:(NSString *)bundleName {
+  // In iOS 8+, we could be included by way of a dynamic framework, and our resource bundles may
+  // not be in the main .app bundle, but rather in a nested framework, so figure out where we live
+  // and use that as the search location.
+  NSBundle *bundle = [NSBundle bundleForClass:[self class]];
+  NSString *resourcePath = [(nil == bundle ? [NSBundle mainBundle] : bundle)resourcePath];
+  return [resourcePath stringByAppendingPathComponent:bundleName];
+}
+
+#pragma mark - Accessibility
+
+- (BOOL)isAccessibilityElement {
+  return YES;
+}
+
+- (NSString *)accessibilityLabel {
+
+  if (self.isAnimating) {
+    if (self.indicatorMode == MDCActivityIndicatorModeIndeterminate) {
+      NSString *key =
+      kMaterialActivityIndicatorStringTable[kStr_MaterialActivityIndicatorInProgressAccessibilityLabel];
+      return NSLocalizedStringFromTableInBundle(key,
```

Comment:
I think the dynamic label is great and we may want to mark the setter as not available, or something else (NSAssert) to let downstream dependencies know that setting the accessibilityLabel will have no effect.  Maybe we want to provide these as default values that clients can override.